index.html

```diff
@@ -20,71 +20,223 @@
+            </ul>
+          </div>
+        </div>
+        <hr class="horizontal" />
+        <div class="socials">
+          <a href="https://github.com/itsmraga-hub"
+            ><img src="assets/icons/github.png" alt="github logo"
+          /></a>
+          <a href="https://github.com/itsmraga-hub"
+            ><img src="assets/icons/linkedin.png" alt="linkedin logo" />
+          </a>
+          <a href="">
+            <img src="assets/icons/fingers.png" alt="two fingers held up" />
+          </a>
+          <a href="https://twitter.com/RagaMacharia"
+            ><img src="assets/icons/twitter.png" alt="teitter logo"
+          /></a>
+          <a href="">
+            <img src="assets/icons/M.png" alt="letter M" />
+          </a>
+        </div>
+        <div class="nav-arrow">
+          <a href="#work"
+            ><img class="arrow bow" src="assets/projects/arrow.png" alt="" />
+          </a>
+          <a href="#work" class="works"
+            ><p class="work roboto align-center">Work</p></a
+          >
+        </div>
+      </section>
+      <section class="grid-container">
+        <div class="grid-item">
+          <h3 class="head-color roboto pr-title" id="work">MY RECENT WORK</h3>
+        </div>
+        <hr class="horizontal" />
+        <div class="grid-item first">
+          <div class="grid-row-item">
+            <img
+              src="assets/projects/Img-Placeholder.png"
+              alt="Project Image"
+              class="project-image"
+            />
+          </div>
+          <div class="grid-row-item">
+            <h4 class="project-title roboto white">Multi-Post Stories</h4>
+          </div>
+          <div class="white-background"></div>
+          <div class="grid-row-item">
+            <p class="inter text-400 align-left">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required, has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+          </div>
+          <div class="grid-row-item">
+            <ul class="lists">
+              <li class="inter white li-border">css</li>
+              <li class="inter white li-border">html</li>
+              <li class="inter white li-border">bootstrap</li>
+              <li class="inter white">Ruby</li>
+            </ul>
+          </div>
+          <div class="grid-row-item">
+            <button type="button" class="btn btn-primary roboto white">
+              See Project
+            </button>
+          </div>
+        </div>
+        <div class="grid-item second">
+          <div class="grid-row-item">
+            <img
+              src="assets/projects/Img-Placeholder.png"
+              alt="Project Image"
+              class="project-image"
+            />
+          </div>
+          <div class="white-background"></div>
```

Comment:
- [ ] Kindly alter this and all other `<div>` tags used for background decorations. Right now, they're causing overflow from the `body` because the transform translate is causing the `<div>` to exceed the body width as you can see in the images below
![image](https://user-images.githubusercontent.com/90556221/176887730-ad61ab0b-4f93-4475-bf0f-b0ef0bbc71a2.png)
Kindly make sure the background decorations do not cause overflows. Infact, you should not need to use the div's at all if you specify the `background-position` property properly and add the background images to the section instead.